### PR TITLE
Merging path changes

### PR DESCRIPTION
**Pull request by Wisperer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,4 +14,113 @@ testResults
 *.sdf
 *.VC.db
 *.cppcheck
-EU4ToV2/build
\ No newline at end of file
+EU4ToV2/build
+EU4toV2/Makefile
+EU4toV2/CMakeCache.txt
+EU4toV2/Copy_Files.sh
```
Comment (Idhrendur) [inline .gitignore]: Probably shouldn't ignore this one.

File: .gitignore [hunk 2]
```diff
@@ -14,4 +14,113 @@ testResults
...
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/Date.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/LinuxUtils.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/Log.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/NewParserToOldParserConverters.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/Object.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/ParadoxParser8859_15.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/ParadoxParserUTF8.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/ParserHelpers.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/newParser.cpp.o
+EU4toV2/Copy_Files.sh
```
Comment (Idhrendur) [inline .gitignore]: Likewise

Comment (Wisperer) [reply .gitignore]: I thought i fixed that already. Ill get ti it tonight.

File: .gitignore (hunk 1 above)
Comment (Wisperer) [reply .gitignore]: done

File: .gitignore (hunk 2 above)
Comment (Wisperer) [reply .gitignore]: done

**Comment by EcceTestudo** [comment]
tested on two saves, both ran for a month no issues. 
should be good to merge

**Pull request merged**